submissions/onyemeMichael/javaScript/script.js

```diff
@@ -52,4 +53,43 @@ for (let product = 0; product < productList.length; product++) {
 // }
 
 // console.log(str);
-// console.log(str2);
\ No newline at end of file
+// console.log(str2);
+
+// TASk 1
+const diff = 123 - 7;
+var  output = 1;
+
+// // //Using Switch Statement
+while (output < diff) {
```

Comment:
Since you are counting from `1`. I think your condition should be `<=` not `<`. Except maybe you want to count from `0`